Schema generator throws an error on Oracle connector due to recent SLF4J migration

```bash
[INFO] --- plugin:3.6.0-SNAPSHOT:generate-api-spec (generate-connector-metadata) @ debezium-connector-oracle ---
Failed to auto configure default logger context
Reported exception:
ch.qos.logback.core.joran.spi.JoranException: Parser configuration error occurred
        at ch.qos.logback.core.joran.event.SaxEventRecorder.buildSaxParser(SaxEventRecorder.java:89)
        at ch.qos.logback.core.joran.event.SaxEventRecorder.recordEvents(SaxEventRecorder.java:57)
        at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:151)
        at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:110)
        at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:53)
        at ch.qos.logback.classic.util.ContextInitializer.configureByResource(ContextInitializer.java:64)
        at ch.qos.logback.classic.util.ContextInitializer.autoConfig(ContextInitializer.java:134)
        at org.slf4j.impl.StaticLoggerBinder.init(StaticLoggerBinder.java:84)
        at org.slf4j.impl.StaticLoggerBinder.<clinit>(StaticLoggerBinder.java:55)
        at org.slf4j.LoggerFactory.bind(LoggerFactory.java:150)
        at org.slf4j.LoggerFactory.performInitialization(LoggerFactory.java:124)
        at org.slf4j.LoggerFactory.getILoggerFactory(LoggerFactory.java:417)
        at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:362)
        at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:388)
        at io.debezium.schemagenerator.SchemaGenerator.<clinit>(SchemaGenerator.java:39)
Caused by: javax.xml.parsers.ParserConfigurationException: SAX feature 'http://xml.org/sax/features/external-general-entities' not supported.
        at oracle.xml.jaxp.JXSAXParserFactory.setFeature(JXSAXParserFactory.java:272)
        at ch.qos.logback.core.joran.event.SaxEventRecorder.buildSaxParser(SaxEventRecorder.java:82)
        ... 14 more
07:47:05,840 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.xml]
07:47:05,841 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Found resource [logback.xml] at [jar:file:/home/ccranfor/github/debezium/debezium/debezium-schema-generator/target/debezium-schema-generator-3.6.0-SNAPSHOT.jar!/logback.xml]
07:47:05,845 |-INFO in ch.qos.logback.core.joran.spi.ConfigurationWatchList@4b9e13df - URL [jar:file:/home/ccranfor/github/debezium/debezium/debezium-schema-generator/target/debezium-schema-generator-3.6.0-SNAPSHOT.jar!/logback.xml] is not of type file
07:47:05,870 |-ERROR in ch.qos.logback.core.joran.event.SaxEventRecorder@2b98378d - Parser configuration error occurred javax.xml.parsers.ParserConfigurationException: SAX feature 'http://xml.org/sax/features/external-general-entities' not supported.
        at javax.xml.parsers.ParserConfigurationException: SAX feature 'http://xml.org/sax/features/external-general-entities' not supported.
        at      at oracle.xml.jaxp.JXSAXParserFactory.setFeature(JXSAXParserFactory.java:272)
```

--- a/debezium-connector-oracle/pom.xml
+++ b/debezium-connector-oracle/pom.xml
@@ -361,6 +361,11 @@
             <plugin>
                 <groupId>io.debezium</groupId>
                 <artifactId>debezium-schema-generator</artifactId>
+                <configuration>
+                    <jvmArgs>
+                        <arg>-Djavax.xml.parsers.SAXParserFactory=com.sun.org.apache.xerces.internal.jaxp.SAXParserFactoryImpl</arg>
+                    </jvmArgs>
+                </configuration>
             </plugin>
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>

--- a/debezium-schema-generator/src/main/java/io/debezium/schemagenerator/maven/SchemaGeneratorMojo.java
+++ b/debezium-schema-generator/src/main/java/io/debezium/schemagenerator/maven/SchemaGeneratorMojo.java
@@ -63,6 +63,12 @@ public class SchemaGeneratorMojo extends AbstractMojo {
     private String filenameSuffix = "";
 
     /**
+     * Optional JVM arguments to pass to the schema generator process.
+     */
+    @Parameter
+    private List<String> jvmArgs;
+
+    /**
      * Gives access to the Maven project information.
      */
     @Parameter(defaultValue = "${project}", required = true, readonly = true)
@@ -84,7 +90,8 @@ public class SchemaGeneratorMojo extends AbstractMojo {
         String classPath = getClassPath();
 
         try {
-            int result = exec(SchemaGenerator.class.getName(), classPath, Collections.emptyList(),
+            List<String> effectiveJvmArgs = jvmArgs != null ? jvmArgs : Collections.emptyList();
+            int result = exec(SchemaGenerator.class.getName(), classPath, effectiveJvmArgs,
                     Arrays.<String> asList(format, outputDirectory.getAbsolutePath(), String.valueOf(groupDirectoryPerComponent),
                             quoteIfNecessary(filenamePrefix), quoteIfNecessary(filenameSuffix),
                             project.getArtifact().getFile().getAbsolutePath()));

